src/django/oar/urls.py

```diff
@@ -39,6 +39,7 @@
     url(r'^api/contributor-types/', views.all_contributor_types,
         name='all_contributor_types'),
     url(r'^api/countries/', views.all_countries, name='all_countries'),
+    url(r'^api/facilities_count', views.facilities_count, name='facilities_count'),
```

Comment:
Consider moving this under the FacilitiesViewSet, but if we need to keep an explicit URL, we should consider `api/facilities-count`. Hyphens are generally preferred in URLs

Reply:
Ended up moving this under the FacilitiesViewSet.